lib/manager/reducers/status.js

```diff
@@ -71,7 +73,7 @@ const config = (state: StatusState = defaultState, action: Action): StatusState
     case 'UPDATING_SERVER':
     case 'FETCHING_ALL_JOBS':
     case 'REQUEST_RTD_ALERTS':
-      return {...state, message: messages(action.type)}
+      return {...state, saving: true, message: messages(action.type)}
```

Comment:
Add comment regarding what sets `saving` back to `false`. Also, a previous commit was setting `saving` back to `false` but I'm not seeing that, meaning the logic is incomplete.